templates/core/2016-2017.html

```diff
@@ -78,7 +85,7 @@ <h2>{% trans "✨ Highlights ✨" %}</h2>
             <div id="growth" class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-10 col-xs-offset-1">
               <h2>{% trans "Growth 📈" %}</h2>
               <div id="map"></div>
-              <small>{% trans "Map of Django Girls event from July 2014 to July 2017" %}</small>
+              <small>{% trans "Map of Django Girls events from January 2016 to July 2017" %}</small>
```

Comment:
This description was referring to different data than were loaded to the map, so I updated it to match what is pulled from the db.